src/js/components/tooltip/index.js

```diff
@@ -67,10 +67,8 @@ export default Component.extend({
     clearTimeout(this.delayTimeout);
   },
   onShow() {
-    anime({
-      targets: this.getView().el,
-      duration: 1500,
-      opacity: [0, 1],
+    animate(this.getView().el, {
+      opacity: { to: 1, duration: 500 },
```

Comment:
Not sure what's up with this one. After switching to v4, this `1500` duration was slower than before.

`500` seems to match the functionality currently in prod.

Reply:
weird...